app.js

```diff
@@ -0,0 +1,48 @@
+const tool = require('./src/tool');
+
+let formBuilder = [
+
+  // Text Inputs
+  {
+    type: "text",
+    id: "username",
+    required: "true",
+    placeholder: "username"
+  },
+  {
+    type: "password",
+    id: "password",
+    required: "true",
+    placeholder: "password"
+  },
+
+  // Select Inputs
+  {
+    type: "select",
+    id: "gender",
+    required: "true",
+    option1: "male",
+    option2: "female"
+  },
+  {
+    type: "select",
+    id: "team",
+    required: "false",
+    option1: "red",
```

Comment:
What is "option1" and "option2"? I don't see these detailed in your documentation.